### DISCUSSION TITLE
Kompatible Batterien

### DISCUSSION THREAD

**Discussion by haegar33** [hoylabs/OpenDTU-OnBattery#1075] [Q&A]
Hallo!
Ist bestimmt ein FAQ aber ich habe keine passende Doku hier gefunden: Welche Batterien sind denn kompatibel mit dem hier geforderten CAN Bus und Protokoll? Anderes gefragt: welche Batterien sind Pylontech kompatibel? Interessant find ich z.B. die Akkus von Marstek?

**Comment by Manos1966** [comment]
> welche Batterien sind Pylontech kompatibel?

Die Batterien auf deren Produkt-Blatt steht "Pylontech-Protokoll" unter CAN Port

**Comment by haegar33** [comment]
Ok, so wie ich die Doku verstanden habe gibt es mehrere Möglichkeiten wie die openDTU ihre Info über Batterien bekommt. Wenn man über ein Victron MPPT geht braucht man nicht unbedingt noch eine CAN Bus Verbindung?

**Comment by Manos1966** [comment]
Korrekt.

> wie die openDTU ihre Info über Batterien bekommt

Es hängt davon ab, wie viele Infos über die Batterie bekommen willst.

Das MINIMUM, dass die OpenDTU-OnBattery nutzt, ist die Batterie Spannung (Volt)
Die Batterie Spannung kann man sehen: 
- am jeden Eingang des Wechselrichters (der an der Batterie angeschlossen ist)
- am Victron

Die Batterie Spannung (Volt) gibt Information ueber den Ladestand der Batterie.
Jede Batterie hat eine Anzahl Zellen (Cells). 
Abhängig vom Material (LiPoFe, Blei, usw) ,  jede Zelle hat einen bestimmten Spannungs Bereich, der gleichzeitig den Ladestand % der Zelle bedeutet.
Die Summe der Zellen gibt die Spannung der Batterie (12V, 24V, 48V, 51V)

Auf Basis also der Spannung (= gesamt Summe der Spannung der individuellen Zellen) kann man also "genau" nachrechnen, welchen Ladezustand % die Batterie hat. 

OHNE CANbus.

Mehr Infos hier: https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Battery%E2%80%90Tipps#cell-voltage

Wenn du "nur" den Ladestand der Batterie sehen willst, braucht also die OpenDTU-OnBattery nicht unbedingt eine CANbus Verbindung

zur Klarheit: Du siehst nur die Spannung Volt der Batterie und du kannst die Lade-, Entlade-Grenzen auf Basis der Spannung definieren (du muss selber wissen wie viel SOC % die jeweilige Spannung bedeutet).
